cluster/ci/config/prow/config.yaml

```diff
@@ -41,8 +41,8 @@ jenkins_operators:
 
 sinker:
   resync_period: 1h
-  max_prowjob_age: 48h
-  max_pod_age: 12h
+  max_prowjob_age: 7d
+  max_pod_age: 24h
```

Comment:
Why increase pod age? I think we can delete pods asap once the tests are finished.